multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -177,280 +889,118 @@ fn verify_bill_payment() {
+        global_config.exp_final_cons_node_service_fee_balance_minor,
     );
-
     assert_balances(
         &serving_node_1_wallet,
         &blockchain_interface,
-        "100000000000000000000",
-        amount.to_string().as_str(),
+        0,
+        global_config.exp_final_service_fee_balance_serv_node_1_minor,
     );
-
     assert_balances(
         &serving_node_2_wallet,
         &blockchain_interface,
-        "100000000000000000000",
-        amount.to_string().as_str(),
+        0,
+        global_config.exp_final_service_fee_balance_serv_node_2_minor,
     );
-
     assert_balances(
         &serving_node_3_wallet,
         &blockchain_interface,
-        "100000000000000000000",
-        amount.to_string().as_str(),
-    );
-
-    let serving_node_1 = cluster.start_named_real_node(
-        &serving_node_1_name,
-        serving_node_1_index,
-        serving_node_1_config,
-    );
-    let serving_node_2 = cluster.start_named_real_node(
-        &serving_node_2_name,
-        serving_node_2_index,
-        serving_node_2_config,
+        0,
+        global_config.exp_final_service_fee_balance_serv_node_3_minor,
     );
-    let serving_node_3 = cluster.start_named_real_node(
-        &serving_node_3_name,
-        serving_node_3_index,
-        serving_node_3_config,
+    let serving_node_1_attributes = ServingNodeAttributes {
+        name: serving_node_1_name.to_string(),
+        index: serving_node_1_index,
+        config: serving_node_1_config,
+    };
+    let serving_node_2_attributes = ServingNodeAttributes {
+        name: serving_node_2_name.to_string(),
+        index: serving_node_2_index,
+        config: serving_node_2_config,
+    };
+    let serving_node_3_attributes = ServingNodeAttributes {
+        name: serving_node_3_name.to_string(),
+        index: serving_node_3_index,
+        config: serving_node_3_config,
+    };
+    start_serving_nodes_and_run_check(
+        &mut cluster,
+        serving_node_1_attributes,
+        serving_node_2_attributes,
+        serving_node_3_attributes,
+        &global_config,
     );
-    for _ in 0..6 {
-        cluster.start_real_node(
-            NodeStartupConfigBuilder::standard()
-                .chain(Chain::Dev)
-                .neighbor(serving_node_1.node_reference())
-                .neighbor(serving_node_2.node_reference())
-                .neighbor(serving_node_3.node_reference())
-                .build(),
-        );
-    }
-
     test_utils::wait_for(Some(1000), Some(15000), || {
-        if let Some(status) = serving_node_1_receivable_dao.account_status(&contract_owner_wallet) {
-            status.balance_wei == 0
+        if let Some(status) = serving_node_1_receivable_dao.account_status(&consuming_node_wallet) {
+            status.balance_wei
+                == i128::try_from(
+                    global_config.debt_size(NodeByRole::ServNode1)
+                        - global_config.exp_final_service_fee_balance_serv_node_1_minor,
```

Comment:
Maybe we should investigate this condition further. Maybe we'd just like to check against the final value instead of subtracting it.